tokio-process/tests/support/mod.rs

```diff
@@ -19,19 +20,16 @@ pub fn cmd(s: &str) -> Command {
     Command::new(me)
 }
 
-pub fn with_timeout<F: Future>(future: F) -> impl Future<Item = F::Item, Error = F::Error> {
-    Timeout::new(future, Duration::from_secs(3)).map_err(|e| {
-        if e.is_timer() {
-            panic!("failed to register timer");
-        } else if e.is_elapsed() {
-            panic!("timed out")
-        } else {
-            e.into_inner().expect("missing inner error")
+pub fn with_timeout<F: Future>(future: F) -> impl Future<Output = F::Output> {
+    Timeout::new(future, Duration::from_secs(10)).then(|r| {
```

Comment:
Any reason we can't keep the timeout to 3 (or even just 1) seconds here?

It's fine if we have some really slow tests, just curious since keeping this value higher will just make tests slower if something is broken...